main.go

```diff
@@ -41,8 +45,10 @@ func main() {
 	config.SsoSecretPtr = flag.String("sso-secret", "", "SSO secret for origin")
 	config.SsoUriPtr = flag.String("sso-url", "", "SSO endpoint eg: http://discourse.forum.com")
 	config.AllowAllPtr = flag.Bool("allow-all", false, "allow all discourse users (default: admin users only)")
+	config.AllowedGroupsPtr = flag.String("allowed-groups", "", "if set, will only auth if the user is member of at least one specified group. eg: 'admins,build'")
 	config.BasicAuthPtr = flag.String("basic-auth", "", "HTTP Basic authentication credentials to let through directly")
 	config.UsernameHeaderPtr = flag.String("username-header", "Discourse-User-Name", "Request header to pass authenticated username into")
+	config.GroupsHeaderPtr = flag.String("groups-header", "Discourse-User-Groups", "Request header to pass authenticated groups into")
```

Comment:
I can't find anywhere the value of this config option actually gets used.

Reply:
Yes, that part was missing sorry.

Now we are properly writing it to the cookie. I will test this in dev env and see if it's okay later today.